File: pingdart-php/src/Services/SmsService.php
<?php

namespace PingDart\SDK\Services;

use GuzzleHttp\Client;

class SmsService {
    private $http;

    public function __construct(Client $http) {
        $this->http = $http;
    }

    public function sendSms(string $to, string $text, string $templateId = null, string $route = 'pingdart', string $unicode = 'true') {
        try {
            $response = $this->http->post('email/send-sms', [
                'json' => [
                    'to' => $to,
                    'text' => $text,
                    'templateId' => $templateId,
                    'route' => $route,
                    'unicode' => $unicode
                ]
            ]);
            return json_decode($response->getBody()->getContents(), true);
        } catch (\Exception $e) {
            throw new \Exception("SMS sending error: " . $e->getMessage());
        }
    }
}

File: pingdart-python/pingdart_sdk/services/sms_service.py
class SmsService:
    def __init__(self, http_client):
        self.http = http_client

    def send_sms(self, to, text, template_id=None, route="pingdart", unicode="true"):
        try:
            response = self.http.post("/email/send-sms", json={
                "to": to,
                "text": text,
                "templateId": template_id,
                "route": route,
                "unicode": unicode
            })
            response.raise_for_status()
            return response.json()
        except Exception as e:
            raise Exception(f"SMS sending error: {str(e)}")
